conda/cli/activate.py

```diff
@@ -62,7 +81,7 @@ def main():
             print(os.environ['PATH'])
             raise
         paths = []
-        sys.stderr.write("discarding %s from PATH\n" % binpath)
+        sys.stderr.write("discarding %s from PATH\n" % pathlist_to_str(binpath))
```

Comment:
Yes, I think this is going to be a nice improvement.  This happens in Conda's symlink step during installation: https://github.com/msarahan/conda/blob/wulmer-fix/activate-behaviour-on-windows/conda/install.py#L430